etna/search/views.py

```diff
@@ -379,6 +378,12 @@ def form_invalid(self, form):
             if field_name in form.errors:
                 return HttpResponseBadRequest(str(form.errors[field_name]))
 
+        if not self.api_result:
```

Comment:
We can drop this conditional and always do this... the API shouldn't have been queried if the form is invalid.

Reply:
Nice. Fixed.